src/chartkit/themes.js

```diff
@@ -0,0 +1,47 @@
+export const ChartColors = {
+  blue: '#1f9bb6',
+  purple: '#e3429b',
+  gridGrey: '#e7e8ec',
+  axisGrey: '#a9acbd',
+  tickTextGrey: '#343434',
+};
```

Comment:
should we have this in the app's theme? we can then inject the color dependencies to the rest of these objects with a function wrap :)

Reply:
Moved colors into app theme.
Leaving as an import in chart theme for now.
Would like to spend a bit more time figuring out the best way to do chart theming.